<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ingreso_datos_poliza</class>
 <widget class="QDialog" name="ingreso_datos_poliza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ingresar Datos:</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="btnVolver">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmar">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>620</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>901</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 236, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 232, 163);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nro. de Cliente:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de Documento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Datos del Cliente:</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtNumeroCliente">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>99-999999999999</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtTipoDocumento">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtApellido">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Domicilio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtNumeroDocumento">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>99-999999999999</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtDomicilio">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nro. de Documento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>861</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelProvin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Provincia</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLocal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Localidad</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMarca">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Marca de Vehículo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKilometros">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kilómetros realizados por año:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPatente">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Patente del Vehículo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtKilometrosAnio">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>50</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0) ;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelModel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Modelo del Vehículo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAnnoVe">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Año del Vehículo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSumaAs">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Suma Asegurada                         </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtChasis">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0) ;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelChasis">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chasis</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMotor">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0) ;
</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbProvincia">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--- Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbLocalidad">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--- Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbMarcaVehiculo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--- Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbModeloVehiculo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--- Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelHijos">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hijos entre 18 y 30 años:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLimpiar">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(13, 153, 255);
border-radius: 8px;
</string>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAgregar">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(13, 153, 255);
border-radius: 8px;
</string>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSexo">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>310</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbEstadoCivil">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>310</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>280</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sexo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>280</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado civil:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbNumeroSiniestros">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--- Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>240</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Número de siniestros en el ultimo año:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbAnioVehiculo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>--- Seleccione una opción</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtPatente">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0) ;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Medidas de Seguridad</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbGaraje">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>90</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>¿Se guarda en garaje?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbAlarma">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>120</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>¿Tiene alarma?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbRastreo">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>150</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>¿Posee dispostivo de rastreo vehicular?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbTuerca">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>180</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>¿Posee tuercas antirrobo en las cuatro ruedas?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>240</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);</string>
     </property>
     <property name="text">
      <string>(*)</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtCantidadHijos">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtSumaAsegurada">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-40</y>
     <width>1031</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Alta de Póliza</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>670</y>
     <width>1031</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 153, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ingresar Datos:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbProvincia</tabstop>
  <tabstop>cbLocalidad</tabstop>
  <tabstop>cbMarcaVehiculo</tabstop>
  <tabstop>cbModeloVehiculo</tabstop>
  <tabstop>cbAnioVehiculo</tabstop>
  <tabstop>txtMotor</tabstop>
  <tabstop>txtChasis</tabstop>
  <tabstop>txtPatente</tabstop>
  <tabstop>txtKilometrosAnio</tabstop>
  <tabstop>chbGaraje</tabstop>
  <tabstop>chbAlarma</tabstop>
  <tabstop>chbRastreo</tabstop>
  <tabstop>chbTuerca</tabstop>
  <tabstop>cbNumeroSiniestros</tabstop>
  <tabstop>cbSexo</tabstop>
  <tabstop>cbEstadoCivil</tabstop>
  <tabstop>btnAgregar</tabstop>
  <tabstop>btnLimpiar</tabstop>
  <tabstop>btnVolver</tabstop>
  <tabstop>btnConfirmar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
